tools/PackageBuilder/NativePackage.cs

```diff
@@ -4,7 +4,7 @@ namespace PackageBuilder
 {
     public class NativePackage
     {
-        public static void Build(string version)
+        public static void Build(string version, string linuxX86, string linuxX64, string osx)
```

Comment:
.NET Core does not support 32-bit Linux, so maybe you don't need linuxX86 at all?